roles/mongo_7_0/defaults/main.yml

```diff
@@ -17,7 +17,7 @@ mongo_log_dir: "{{ COMMON_LOG_DIR }}/mongo"
 mongo_journal_dir: "{{ COMMON_DATA_DIR }}/mongo/mongodb/journal"
 mongo_user: mongodb
 
-MONGODB_REPO: deb http://repo.mongodb.org/apt/ubuntu {{ ansible_distribution_release }}/mongodb-org/{{ MONGO_VERSION_MAJOR_MINOR }} multiverse
+MONGODB_REPO: deb http://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/{{ MONGO_VERSION_MAJOR_MINOR }} multiverse
```

Comment:
why the hardcoded jammy?

Reply:
mongo only have support to mongo8.0 in noble